_posts/2021-01-21-cockpit-236.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Cockpit 236
+author: pitti
+date: 2021-01-07
+tags: cockpit linux
+slug: cockpit-236
+category: release
+summary: Cockpit with SSH key improvements and motd editing
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 236.
+
+## fslist channels: Include properties of changed files
+
+The [fslist channel API](https://github.com/cockpit-project/cockpit/blob/master/doc/protocol.md#payload-fslist1) lists files in a directory, watches for further changes to files, and sends events when changes happen. These notifications now include additional properties about the file: the owner and group name, file size, and time of last modification.
+
+These changes were made with the goal of implementing an efficient file manager Cockpit page.
+
+_Thanks to leomoty for this contribution!_
```

Comment:
Continuing from above, but including both sentences (which is why this breaks the conversation thread a bit).

Perhaps something like this?

```suggestion
This contribution from leomoty provides a backend for efficient file browsing, which could serve as the basis of a third-party file selector or file manager. _Thanks, leomoty!_
```

Reply:
We already have a "file selector", and that is usually a short-term thing.

This contribution from leomoty can serve as the basis of a third-party file manager page. _Thanks leomoty!_

?

Reply:
```suggestion
This contribution from leomoty can serve as the basis of a third-party file manager page. _Thanks, leomoty!_
```

Yeah, sounds good. Adding as a suggestion here, so it's just a click to change. :wink: